### ISSUE TITLE
Options to disable unimportant items for Outline View

### ISSUE THREAD

**Issue by mik3y64** [julia-vscode/julia-vscode#1158]
Thank you for making this vscode extension for Julia. I am using it for my daily driver for running julia scripts.

I use the Outline View all the time but I find the unimportant items such as variables (outside of structs and functions) are clogging up the outline views making it practically unusable. In some of my scripts, I have more than 50 variables and I just could not use Outline view to see my important functions and structs. For instsance,

```julia
x = "a"
y = 356
z = [3, 4]

struct MyType
    num::Interger
end

function MyFunction(xx, yy)
    xx + yy
end

x2 = "b"
y2 = 789
z2 = [300, 400]
````

Outline View gets clogged up by all the variables, making it unusable.

```julia
x
y
z
MyType
    num
MyFunction
    xx
    yy
x2
y2
z2

# When you have many variables, the unimportant variables keep growing and growing
# x3
# y3
# z3
# MyFunction2
# x4
# y4
# z4
# MyType2
# x5
# y5
# z5
# MyFunction3
# x6
# y6
# z6
# MyType3
# x7
# y7
# z7
````

What I want to see

```julia
MyType
    num
MyFunction
    xx
    yy
```

Some vscode options `"outline.showVariables": false` are not optimal. It only hides some noisy variables but doesn't hide all and it hides all variables within structs and functions which I need them.

```julia
x
y
MyType
MyFunc
```

**Comment by davidanthoff** [comment]
I agree, but I think this probably needs to be solved upstream... We only provide all the symbols to VS Code, and VS Code then decides how to display them.

We could not report certain symbols, but that would affect many other parts of the editing experience, so I don't think that would be a good idea.
